### PR TITLE
Fixed Broken Link in README

### LINKED ISSUE #1876: [Docs] Broken Link
**Current State:**

The **Service Mesh Day** link is broken in README file

**Desired State:**

Update it to https://servicemeshday.com/schedule.html

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by SaravgiYash**
**Description**
Broken link fixed in readme
This PR fixes #1876

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by akashgp09** [comment]
@Yashs911 Please Sign off your commit.

**Comment by tripathyprateek** [comment]
https://github.com/layer5io/meshery/pull/1878/checks?check_run_id=1370375587 @Yashs911 check this link to signoff your commits.

**Comment by Yashs911** [comment]
@tripathyprateek When I run git commit --amend --signoff it opens a git interactive window

File: docs/pages/index.html [hunk 1]
```diff
@@ -40,6 +40,14 @@
     <a href="{{ site.baseurl }}/concepts">
         <div class="overview">Concepts</div>
     </a>
+    <ul><b>Concepts</b>
```
Comment (leecalcote) [inline docs/pages/index.html:43]: This set of changes is from a different collection of commits. Ideally, this set of changes is not included in this PR.

Not a big deal in this case, so we're ok.

File: README.md [hunk 2]
```diff
@@ -245,7 +245,7 @@ We're a warm and welcoming community of open source contributors. Please join. A
...
 - [DockerCon 2019 Open Source Summit](https://dockercon19.smarteventscloud.com/connect/sessionDetail.ww?SESSION_ID=309149&tclass=popup#.XJxH-TOcbjI.twitter) | [deck](https://calcotestudios.com/talks/decks/slides-dockercon-2019-establishing-an-open-source-office.html), [video](https://www.docker.com/dockercon/2019-videos?watch=open-source-summit-service-mesh)
 - [Container World 2019](https://tmt.knect365.com/container-world/speakers/lee-calcote) | [deck](https://calcotestudios.com/talks/decks/slides-container-world-2019-service-meshes-but-at-what-cost.html)
-- [Service Mesh Day](https://servicemeshday.com/schedule/) | [deck](https://docs.google.com/presentation/d/1HwG03okX3DHgGKbma4PL-MO7Xr9zDrjQgd05PRi9i8E/edit?usp=sharing), [video](https://youtu.be/CFj1O_uyhhs)
+- [Service Mesh Day](https://servicemeshday.com/schedule.html) | [deck](https://docs.google.com/presentation/d/1HwG03okX3DHgGKbma4PL-MO7Xr9zDrjQgd05PRi9i8E/edit?usp=sharing), [video](https://youtu.be/CFj1O_uyhhs)
```
Comment (leecalcote) [inline README.md:248]: Looks good.

**Review by leecalcote — APPROVED** [review]
Thank you, @Yashs911 and @akashgp09

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Yay, @Yashs911 ! 👍

**Pull request merged**